########################################
# Actuator
########################################

uint32 type # Actuator type, refer to rtabmap::Actuator::Type enum)
rtabmap/CvMatMsg matrix